From: Andrew Morton <akpm@linux-foundation.org>
Subject: shmem-prepare-shmem-quota-infrastructure-fix
Date: Fri Jul 14 08:54:52 AM PDT 2023

make shmem_mark_dquot_dirty and shmem_dquot_write_info static

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202307141341.R961dyMt-lkp@intel.com/
Cc: Carlos Maiolino <cem@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/shmem_quota.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/shmem_quota.c~shmem-prepare-shmem-quota-infrastructure-fix
+++ a/mm/shmem_quota.c
@@ -283,12 +283,12 @@ out_dqlock:
 	return 0;
 }
 
-int shmem_mark_dquot_dirty(struct dquot *dquot)
+static int shmem_mark_dquot_dirty(struct dquot *dquot)
 {
 	return 0;
 }
 
-int shmem_dquot_write_info(struct super_block *sb, int type)
+static int shmem_dquot_write_info(struct super_block *sb, int type)
 {
 	return 0;
 }
_
